packages/contentful--app-scripts/lib/get-management-token.js

```diff
@@ -5,30 +5,27 @@ const open = require('open');
 const inquirer = require('inquirer');
 const { cacheEnvVars } = require('../utils/cache-credential');
 const { createClient } = require('contentful-management');
-const {
-  ACCESS_TOKEN_ENV_KEY
-} = require('../utils/constants');
+const { ACCESS_TOKEN_ENV_KEY } = require('../utils/constants');
 
-const checkTokenValidity = async (accessToken) => {
+const checkTokenValidity = async (accessToken, host) => {
   try {
-    const client = createClient({ accessToken });
+    const client = createClient({ accessToken, host });
     await client.getCurrentUser();
     return true;
-
-  } catch(err) {
+  } catch (err) {
     return false;
   }
-}
+};
 
-async function getManagementToken() {
+async function getManagementToken(host = undefined) {
```

Comment:
one small question, why to specifically define `undefined` as by default parameters have `undefined` value in javascript.

Reply:
sematics; it's an optional argument.. in Typescript i'd use `?` which is way more clear, but I've removed it